_posts/2016-08-17-improve-teams-communication-with-user-stories.md

```diff
@@ -0,0 +1,88 @@
+---
+author: Giulia Mantuano
+comments: true
+date: 2016-08-17 14:25:09+00:00
+layout: post
+asset-type: post
+slug: improve-teams-communication-with-user-stories
+title: Improve team's communication with User Stories
+image:
+    src: /assets/img/custom/blog/user-stories-sketchnote.jpg
+category: agile
+tags:
+- scrum
+- user stories
+- team
+---
+
+The challenge of Software development is about building a product which fulfills both business and users expectations. 
+
+The focus for each role within an Agile team is different: Project Managers want to see progress and want a quality product that is in time and value for money; Product Manager wants flexibility; Testers want to measure; Developers wants to write outstanding and sustainable code; Users are looking for a product which helps accomplish their tasks. How is it possible to find an agreement between all of these needs and build a product which could be a good compromise between building the right thing, building the thing right and building it in time?
```

Comment:
Product Manager wants flexibility; ==> Product Managers want flexibility;

(try to keep it singular or plural, not mixing the number)